metals/src/main/scala/scala/meta/internal/metals/FileDecoderProvider.scala

```diff
@@ -67,29 +67,27 @@ final class FileDecoderProvider(
    * jar:file:///jarPath/jar-sources.jar!/packagedir/file.java
    *
    * Examples...
-   * metalsDecode:file:///somePath/someFile.java.javap-verbose?decoder=javap&verbose=true
```

Comment:
We should still be able to run on a java file.

Reply:
What about `metalsDecode:file:///somePath/someFile.java.semanticdb-compact`, is this correct? I thought that semanticDB could be produced only from scala files.

Reply:
I went with the first option because it requires only a few minor changes in implementation. I like the idea of `goto-location`, but this would require more changes in metals-vscode and scope of this and vscode related PR are big enough for now.

Reply:
> I thought that semanticDB could be produced only from scala files.

There were some recent changes in Bloop utilizing `lsif-java` to produce semanticDB for java files as well. https://github.com/scalacenter/bloop/pull/1534